RecoTracker/LSTCore/interface/LST.h

```diff
@@ -16,14 +16,15 @@ namespace SDL {
   template <typename>
```

Comment:
is `#include <alpaka/alpaka.hpp>` on L10 allowed for code in non-alpaka builds?

Reply:
Yeah, anything that doesn't need `ALPAKA_ACCELERATOR_NAMESPACE` should be placed outside.